1/assignment-week1.py

```diff
@@ -0,0 +1,83 @@
+print ("Week 1 Assignment - Math Equation Evaluator")
+print ("Intern - Shu Wei")
+print ("")
+
+eArray = list(map(str, input("Please enter your input with spaces between them: ").split()))

```

Comment:
no need to call `str` your input split will already be strings

Reply:
in python you probably dont need to call list. there are only a few cases you need to call list after a map. we might cover that later